sympy/integrals/tests/test_integrals.py

```diff
@@ -1388,3 +1388,9 @@ def test_issue_14782():
 def test_issue_12081():
     f = x**(-S(3)/2)*exp(-x)
     assert integrate(f, [x, 0, oo]) == oo
+
+
+def test_issue_15285():
+    y = 1/x - 1
+    f = (4*y*exp(-2*y))/(x**2)
```

Comment:
Just `f=4*y*exp(-2*y)/x**2` should suffice.

Reply:
with a space on either side of `=`.